sites/partners/__tests__/components/listings/PaperListingForm/sections/CommunityType.test.tsx

```diff
@@ -154,14 +154,12 @@ describe("CommunityType", () => {
     await act(() => userEvent.click(screen.getByRole("radio", { name: "Yes" })))
 
     expect(
-      screen.getByRole("textbox", { name: "Reserved Community Disclaimer Title" })
+      screen.getByRole("textbox", { name: "Reserved Community Disclaimer Title *" })
```

Comment:
question: Is there an easy way to write a test that covers adding the asterisk to all of the fields? I know the listing form doesn't have full test coverage so it could be a lot of work.